apps/web/tests/test_superuser_view.py

```
@@ -0,0 +1,87 @@
+import pytest
+from django.urls import reverse
+from pytest_django.asserts import assertFormError, assertRedirects
+
+from apps.utils.factories.team import TeamFactory
+from apps.utils.factories.user import UserFactory
+from apps.web.views import ADMIN_SLUG
+
+
+@pytest.fixture()
```

Ruff (0.8.0)

Use `@pytest.fixture` over `@pytest.fixture()`

Remove parentheses

(PT001)